src/test/scala/nl/knaw/dans/easy/multideposit/BlackBoxSpec.scala

```diff
@@ -125,4 +128,13 @@ class BlackBoxSpec extends UnitSpec with BeforeAndAfter with MockFactory {
         )
     }
   }
+
+  def transformer(label: String) = new RuleTransformer(new RewriteRule {
```

Comment:
Does it need to be called `transformer`? That doesn't convey what it does. It seems to be deleting all the elements named `"$label"`. Is that correct? If so, maybe `removeElementsByName` or something, would better fit the bill.